### DISCUSSION TITLE
Project still maintained?

### DISCUSSION THREAD

**Discussion by cpotey** [pacocoursey/cmdk#190] [General]
Just curious if this is project is still actively maintained? I see there's a few PRs flying about, and not much movement on things that would help push it further - [asChild props](https://github.com/pacocoursey/cmdk/pull/138) etc

Not attempting to throw any shade towards the developers, as the work is fantastic, and so many of us use this - but it feels like it's become somewhat stagnant whilst there's a growing amount of new users from publicity via shadcn.

**Comment by goleary** [comment]
Beyond PRs, there are changes that have been merged already, but not released (like `forceMount`).

**Comment by dlvhdr** [comment]
I would like to use this for my company's product and wondering the same thing. 
What alternatives are people using if this turns out not to be maintained?

**Comment by pacocoursey** [comment]
Sorry for the delay. A new release is out, and I'll be maintaining this package more regularly now.
